src/iexamine.cpp

```diff
@@ -2969,7 +2969,7 @@ void iexamine::pay_gas(player *p, map *m, const tripoint &examp)
     int pricePerUnit = getPricePerGasUnit(discount);
     std::string unitPriceStr = string_format(_("$%0.2f"), pricePerUnit / 100.0);
 
-    bool can_hack = (!p->has_trait("ILLITERATE") && ((p->has_amount("electrohack", 1)) ||
+    bool can_hack = (!p->has_trait("ILLITERATE") && ((p->has_charges("electrohack", 10)) ||
                      (p->has_bionic("bio_fingerhack") && p->power_level > 0)));
```

Comment:
Not really related to electrohack, but here the fingerhack should require 25 energy